lib/Runtime/Library/JavascriptError.cpp

```diff
@@ -371,7 +371,9 @@ namespace Js
 
         if (FACILITY_CONTROL == HRESULT_FACILITY(hr) || FACILITY_JSCRIPT == HRESULT_FACILITY(hr))
         {
+#ifndef __ANDROID__
```

Comment:
Actually, shouldn't we consider argList will never be null and keep it as is here?